### ISSUE TITLE
Yes, you have added an input box to it. But the input of the user and the output which ChatGPT gave are correct (but incomplete) and the continue box which you have added is not connected to previous output which ChatGPT gave. So, if i ask any question in the continue box it's like asking a new question to ChatGPT. I'll attach an image for you reference

### ISSUE THREAD

**Issue by nithinreddyy** [hunkimForks/chatgpt-arxiv-extension#19]
Yes, you have added an input box to it. But the input of the user and the output which ChatGPT gave are correct (but incomplete) and the continue box which you have added is not connected to previous output which ChatGPT gave. So, if i ask any question in the continue box it's like asking a new question to ChatGPT. I'll attach an image for you reference

![image](https://user-images.githubusercontent.com/56256685/219961573-7fa9d9b8-4c5c-4151-b685-63a7b82e5e4e.png)

Can you please look into this as this is one of the key issues?

_Originally posted by @nithinreddyy in https://github.com/hunkimForks/chatgpt-arxiv-extension/issues/12#issuecomment-1436033663_

**Comment by hunkim** [comment]
Yes. It's a known issue. I have passed message_id and parent_id, but I will test more. @ yanggak12 Could you take a look?

**Comment by nithinreddyy** [comment]
> Yes. It's a known issue. I have passed message_id and parent_id, but I will test more. @ yanggak12 Could you take a look?

Is it working now?

**Comment by nithinreddyy** [comment]
Hello, any update on this?

**Comment by ishandutta2007** [comment]
Addressed in PR https://github.com/hunkimForks/chatgpt-arxiv-extension/pull/31
